src/App.vue

```diff
@@ -141,6 +144,27 @@ export default {
       await Promise.all(Object.entries(tokens).map(async ([token]) => this
         .addTokenBalance({ token, balance: await tokenBalance(token, address) })));
     },
+    async getCookiesList(address) {
+      await this.backendAuth('getCookiesList', address)
+        .then((list) => list.forEach((entry) => this.setCookiesListEntry(entry)));
+    },
+    // eslint-disable-next-line consistent-return
+    async backendAuth(method, address) {
+      const { challenge } = await Backend[method](address);
+      if (this.useSdkWallet) {
+        const signature = await client.signMessage(challenge);
+        const response = await Backend[method](address, { challenge, signature });
+        return response;
```

Comment:
Actually, you can implement it in BackendAuthMixin

Reply:
currently, this method is moved to https://github.com/aeternity/superhero-ui/blob/9abcbdf26c711dd70b01ca433682ff01e3a7ef50/src/store/modules/backend.js#L115-L130, you don't need to use BackendAuthMixin because we won't use deeplinks this time